packages/data/src/settings/selectors.js

```diff
@@ -18,6 +18,30 @@ export const getSettings = ( state, group ) => {
 	if ( settingIds.length === 0 ) {
 		return settings;
 	}
+
+	settingIds.forEach( ( id ) => {
+		settings[ id ] = getSettingValues(
+			state[ getResourceName( group, id ) ].data
+		);
+	} );
+	return settings;
+};
+
+const getSettingValues = ( settings ) => {
+	return settings.reduce( ( resource, setting ) => {
```

Comment:
Ah, this is related to the general settings I mentioned above.  Was hoping to tackle in a follow-up, but since this portion is not behind a feature flag, I updated it to preload all general settings for this PR and may attempt to reduce some of this preloaded data in a follow-up.

>Address insufficient preloading of "General" settings

Reply:
Aaah sorry, I mis understood, I thought you were referring to the `General` settings page.
Thanks for fixing it anyway though :) given this is being merged into `add/settings-client` I assume it's not going into `main` just yet?